LICENSE.txt

```diff
@@ -1,4 +1,4 @@
-DYNAMO v.2.15.0 © 2022 Autodesk, Inc.  All rights reserved.
+DYNAMO v.2.16.1 © 2022 Autodesk, Inc.  All rights reserved.
```

Comment:
I thought this was bumped up already in the 2.16 release branch 😕

Reply:
I did the about box content but must have missed this file - it doesn't make it into the application so it's not an issue for users IMO.